src/features/left-sidebar/defaultNavLinkRenderer.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import * as React from 'react';
+import LeftSidebarLink from './LeftSidebarLink';
+
+import type { Callout } from './Callout';
+
+type Props = {
+    callout?: Callout,
+    className?: string,
+    customTheme?: Object,
+    htmlAttributes?: Object,
```

Comment:
Would be nice if we can change these Objects to actual { something } shape

Reply:
How do you suggest declaring html attributes as type? Not sure if you meant to list out everything in https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes.